locale/en/docs/guides/getting-started-guide.md

```diff
@@ -21,7 +21,7 @@ const server = http.createServer((req, res) => {
 });
 
 server.listen(port, hostname, () => {
-  console.log(`Server running at http://${hostname}:${port}/`);
+  console.log(`Server running at http://%s:%d/', hostname, port);
```

Comment:
This introduces a syntax error. This has changed one of the backquotes to a single quote. In other words, this breaks the template string.